prediction-insights.ipynb

```diff
@@ -0,0 +1,774 @@
+    "        \"feature_contributions\": True,\n",
+    "        \"feature_residuals\": True,\n",
+    "        \"robust_influences\": True,\n",
+    "        \"robust_residuals\": True,\n",
+    "        \"local_case_feature_residual_convictions\": True,\n",
+    "        \"categorical_action_probabilities\": True,\n",
+    "    }\n",
+    ")\n",
+    "influential_cases = pd.DataFrame(reaction[\"details\"][\"influential_cases\"][0])\n",
+    "influential_cases = influential_cases[sorted(influential_cases.columns)]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Insights\n",
+    "\n",
+    "### Insight 1: Prediction\n",
+    "\n",
+    "The predicted value(s) are accessed using the `action` key, while the explanations can be accessed using the\n",
+    "`details` key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['influential_cases',\n",
+       " 'feature_contributions',\n",
+       " 'action_features',\n",
+       " 'feature_residuals',\n",
+       " 'local_case_feature_residual_convictions',\n",
+       " 'similarity_conviction',\n",
+       " 'categorical_action_probabilities',\n",
+       " 'directional_feature_contributions']"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "list(reaction[\"details\"].keys())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>target</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   target\n",
+       "0       1"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "reaction[\"action\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For categorical action features, the prediction can be further understood with the `categorical_action_probabilities`\n",
+    "detail."
```

Comment:
What is the categorical action probability? In the playground we call this the "confidence" - can we explain to people what this means? For example: We have predicted the iris type to be xyz, with a confidence of 90%. This indicates that...